### PR TITLE
compare correctly to file size in step-06.js

### LINKED ISSUE #112: promises: step-06.js
I can run steps 1-5 successfully but `node step-06.js  .` returns `lengths []`

### PR DESCRIPTION
**Pull request by tobyhodges**
Fixes #112

### CODE REVIEW

File: src/promises/step-06.js [hunk 1]
```diff
@@ -15,9 +15,9 @@ const countLines = (text) => {
   return text.split('\n').length
 }
 
-glob(`${srcDir}/**/*.txt`)
+glob(`${srcDir}/*.txt`)
```
Comment (gvwilson) [inline src/promises/step-06.js]: This change means that the `glob` only matches files directly in `srcDir`. Using `/**/*.txt` finds all files recursively.

File: src/promises/step-06.js [hunk 2]
```diff
@@ -15,9 +15,9 @@ const countLines = (text) => {
...
   .then(files => Promise.all(files.map(f => statPair(f))))
-  .then(files => files.filter(pair => pair.stats > 100000))
+  .then(files => files.filter(pair => pair.stats.size > 100000))
```
Comment (gvwilson) [inline src/promises/step-06.js]: :+1:.

Comment (gvwilson) [inline src/promises/step-06.js]: Please make the corresponding change in promises.tex (I think on line 593).

**Comment by tobyhodges** [comment]
Ready

**Pull request merged**